src/reports/components/instance-details.scss

```diff
@@ -27,7 +17,6 @@
     outline-style: 'border-style';
     box-shadow: 0 0.6px 1.8px $box-shadow-108, 0 3.2px 7.2px $box-shadow-132;
     margin-bottom: 16px;
-    @include fill-available-width;
```

Comment:
Was it intentional to remove this entirely rather than replacing it with `width: fill-available;`?

Reply:
When I tested initially, I found no impact from this declaration. I'm not quite finished validating these changes but will share my findings when I'm ready for review.

Reply:
From testing the extension and generated report, I don't believe setting `width` has an impact on this element and appears safe to remove. Please let me know if there's a scenario I'm missing.